pages/_document.tsx

```diff
@@ -17,11 +17,11 @@ export default function Document() {
         <meta name="yandex-verification" content="d7c2b9f9a4891879" />
       </Head>
       <title>Курсы фронтенд разработчика бесплатно с нуля</title>
-      <meta name="description" content="Обучение фронтенд разработчике бесплатно! Курсы для тех, кто начинает с нуля или хочет улучшить свой код. Научитесь разворачивать фронтенд в облаке, тестировать, делать фронтенд независимый от бекенда." />
+      <meta name="description" content="Обучение фронтенд разработке бесплатно! Курсы для тех, кто начинает с нуля или хочет улучшить свой код. Научитесь разворачивать фронтенд в облаке, тестировать, делать фронтенд независимый от бекенда." />
       <body>
         <Main />
         <NextScript />
-        {process.env.HOST === 'localhost:3000'
+        {process.env.HOST !== 'localhost:3000'
```

Comment:
and if there is a localhost 3001, then the metric will work. You need to check that the host has the word localhost and do not get tied to the port